cpu/link.x

```diff
@@ -1,4 +1,4 @@
-SECTIONS 
```

Comment:
Please move all the reformatting changes to a separate commit in this PR so it's easier to notice the important stuff.